### PR TITLE
Mark Repository as Read

### LINKED ISSUE #60: Mark repo as read
Add a button to mark all notifications for a repo as read.

### PR DESCRIPTION
**Pull request by ekonstantinidis**
Closes #60.

screen shot 2015-06-20 at 20 24 56

 
screen shot 2015-06-20 at 20 44 09

screen shot 2015-06-20 at 20 40 00

### CODE REVIEW

**Comment by ekonstantinidis** [comment]
@JakeSidSmith Wanna review this one? Works great & coverage as always - 100%!

File: src/js/components/repository.js [hunk 1]
```diff
@@ -15,7 +22,29 @@ var Repository = React.createClass({
...
+      .putAuth('https://api.github.com/repos/' + loginId + '/' + repoId + '/notifications', {})
+      .end(function (err, response) {
+        if (response && response.ok) {
+          // Notification Read
+          self.setState({
+            isRead: true
+          });
+        } else {
+          // Error - Show messages.
+          // Show appropriate message
```
Comment (JakeSidSmith) [inline src/js/components/repository.js]: We should probably add a nice way to display errors.

**Comment by JakeSidSmith** [comment]
Could you add a couple of screenshots? :)

**Comment by ekonstantinidis** [comment]
@JakeSidSmith Ready? :rocket:

File: src/js/components/repository.js [hunk 2]
```diff
@@ -15,7 +23,32 @@ var Repository = React.createClass({
...
+    var repoId = this.props.repo[0].repository.name;
+
+    apiRequests
+      .putAuth('https://api.github.com/repos/' + loginId + '/' + repoId + '/notifications', {})
+      .end(function (err, response) {
+        if (response && response.ok) {
+          // Notification Read
+          self.setState({
+            isRead: true,
+            erros: false
```
Comment (JakeSidSmith) [inline src/js/components/repository.js]: Typo. :)

**Comment by JakeSidSmith** [comment]
:rocket:

**Pull request merged**